packages/genesys-spark-components/src/components/stable/gux-listbox-multi/gux-option-multi/gux-option-multi.tsx

```diff
@@ -33,6 +33,9 @@ export class GuxOptionMulti {
   @Prop()
   value: string;
 
+  @Prop()
+  subtext: string;
```

Comment:
We typically don't use props to get text into the dom of componentes because we have found that nearly every time we do we run into the case where devs want to include an icon or special markup etc... and this stops them doing it.
See can you make an optional `subtext` slot work.

Reply:
Thanks, that's really handy to know. I've updated the branch to use a slot instead.